wsmaster/che-core-api-workspace/src/main/resources/schema/1.0.0/devfile.json

```diff
@@ -617,6 +617,7 @@
       "description": "Description of the predefined commands to be available in workspace",
       "items": {
         "type": "object",
+        "additionalProperties": false,
```

Comment:
why do we need ``` "additionalProperties": false,```?

Reply:
Good catch. Without that, we accept devfile `1.0.0` with `previewUrl` (or any extra property). This prevents it and enforce to have `1.0.1-beta` when `previewUrl` defined.
I'm not sure if it is a good idea and we do not need that for general preview url functionality. It just makes validation more strict.